plugins/experimental/ts_lua/ts_lua_misc.c

```diff
@@ -102,11 +102,25 @@ ts_lua_get_now_time(lua_State *L)
 static int
 ts_lua_debug(lua_State *L)
 {
+#define MAX_FLAG_LEN 20
   const char *msg;
-  size_t len = 0;
+  const char *flag;
+  char flags_buffer[MAX_FLAG_LEN];
+  size_t msg_len = 0, flag_len = 0;
+
+  if (lua_gettop(L) == 2) {
+    msg   = luaL_checklstring(L, 2, &msg_len);
+    flag  = luaL_checklstring(L, 1, &flag_len);
+    int n = flag_len + 1 > MAX_FLAG_LEN - 1 ? MAX_FLAG_LEN - 1 : flag_len + 1;
+    memcpy(flags_buffer, flag, n);
+    flags_buffer[n] = 0;
+  } else {
+    msg = luaL_checklstring(L, 1, &msg_len);
+    memcpy(flags_buffer, TS_LUA_DEBUG_TAG, 6);
```

Comment:
Ummmm! It seems that Lua doesn't promise the string alway with '\0' ending(Lua Document), so I try to copy it.
Any idea to fix it ?